app/helpers/emoji_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module EmojiHelper
+  def emoji_codes_to_img(str)
```

Comment:
Small suggestion, but I find it better when we do the short-circuits at the beginning of the method.
Plus, it reduces the nesting level 👍 

``` ruby
def emoji_codes_to_img(str)
  return if str.blank?

  raw str.gsub(/foobar/) do |name|
    # ...
  end
end
```